databus/src/test/java/com/bazaarvoice/emodb/databus/core/DefaultFanoutTest.java

```diff
@@ -80,12 +79,12 @@ public Void apply(Multimap<String, ByteBuffer> input) {
         };
 
         // Event event keys are deleted we need to capture them since fanout clears and re-uses the same instance.
-        _eventSource = mock(EventSource.class);
+        EventSource eventSource = mock(EventSource.class);
```

Comment:
I wouldn't change the naming convention for a single variable, considering the rest is still using underscore. That's not the point of PR.

Reply:
it's not naming only, variable is used only in `setUp()` method, so it doesn't make sense to keep it as class variable.
It's probably not the point of PR, but i would leave it, as change is already done.

Reply:
reverted